pycipapi/cipapi_client.py

```diff
@@ -12,6 +12,7 @@ class CipApiClient(RestClient):
     IG_ENDPOINT = "{url_base}/interpreted-genome".format(url_base=ENDPOINT_BASE)
     REFERRAL_ENDPOINT = "{url_base}/referral".format(url_base=ENDPOINT_BASE)
     FILE_ENDPOINT = "{url_base}/file".format(url_base=ENDPOINT_BASE)
+    AF_ENDPOINT = "{url_base}/participants".format(url_base=ENDPOINT_BASE)
```

Comment:
change name? PARTICIPANTS_ENDPOINT?
after all, we're going to use it now for additional findings, but it's a generic thing not exclusive to additional findings